stock_quant_product_packaging/models/stock_quant.py

```diff
@@ -0,0 +1,27 @@
+# Copyright 2024 Akretion
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class StockQuant(models.Model):
+    _inherit = "stock.quant"
+
+    packaging_info_ids = fields.One2many(
+        "stock.quant.packaging.info",
+        "quant_id",
+        string="Packaging additional information",
+    )
```

Comment:
Is this persistent? My understanding is that Odoo can randomly decide to [merge](https://github.com/OCA/OCB/blob/16.0/addons/stock/models/stock_quant.py#L939) quants that look similar, and sum the quantities into one total, and then you will lose your packaging information.

If you don't want that, you have to make at least one of these fields unique: `GROUP BY product_id, company_id, location_id, lot_id, package_id, owner_id`.

Funnily, that actually contains `package_id`, so maybe you can make use of that. I don't know what it does, though.

Another problem might be that once your stock on hand drops to zero, your quant is [unlinked](https://github.com/OCA/OCB/blob/16.0/addons/stock/models/stock_quant.py#L929). If stock comes in again, a quant will be created, but you'll need to somehow add it to the right package by default. Did you add code for that?